cgmes/cgmes-model/src/main/java/com/powsybl/cgmes/model/CgmesNamespace.java

```diff
@@ -79,4 +80,8 @@ public static String getProfile(int cimVersion, String profile) {
         }
         throw new AssertionError("Unsupported CIM version " + cimVersion);
     }
+
+    public static boolean isEqCgmes3OrGreater(String profile) {
+        return profile.startsWith(CGMES_EQ_3_OR_GREATER_PREFIX) && profile.compareTo(CIM_100_EQ_PROFILE) >= 0;
+    }
```

Comment:
I really don't like this kind of really specific method in a final class... If this is only used once, you can move this method in CgmesModelTripleStore.